src/__internal__/dom/globals.server.test.ts

```diff
@@ -0,0 +1,28 @@
+import { getWindow, getDocument, getNavigator } from "./globals";
+
+describe("getWindow on server", () => {
```

Comment:
comment(non-blocking): for test consistency we have moved away from single test describe blocks

We will likely just tidy this up for consistency once it's in though, so don't worry about it just flagging for future reference